Modix.Services/Utilities/ApplyChannelLimitsAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Discord.Commands;
+using Modix.Data;
+using Modix.Data.Utilities;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Modix.Services.Utilities
+{
+    public class ApplyChannelLimits : PreconditionAttribute
+    {
+        public override Task<PreconditionResult> CheckPermissions(ICommandContext context, CommandInfo command, IServiceProvider map)
+        {
+            using (var db = map.GetService<ModixContext>())
+            {
```

Comment:
are we fine making a database query every single time a command is issued? Can/should this be cached?